### PR TITLE
Adds roundend nuke disk location tracking to feedback.

### PR DESCRIPTION
**Pull request by AnturK**
@nfreader

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by nfreader** [comment]
BETTER EXPERIENCES THROUGH DATA

**Comment by optimumtact** [comment]
won't it be in the nuke

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -63,6 +66,25 @@
...
+		var/turf/T = get_turf(N)
+		if(T)
+			data["x"] = T.x
+			data["y"] = T.y
+			data["z"] = T.z
+		var/atom/outer = get_atom_on_turf(N)
+		if(outer != N)
+			if(isliving(outer))
+				var/mob/living/L = outer
+				.["holder"] = L.real_name
```
Comment (Jordie0608) [inline code/__HELPERS/roundend.dm]: Shouldn't this be `data["holder"]`?

Comment (AnturK) [reply code/__HELPERS/roundend.dm]: Oh yes, whoops.

**Comment by AnturK** [comment]
Only in the relatively small number of rounds, this is to track captain's keeping hold of the disk in general.

**Comment by YotaXP** [comment]
Looks like a really sketchy method of finding the "holder".

**Comment by AnturK** [comment]
Captain hiding in a locker is not captain of mine.

**Comment by YotaXP** [comment]
What if the disk is just lying on the ground, and someone happens to be standing over it?  Or maybe it's on the captain's dead body, but there are multiple bodies on the same turf?  There must be a better way to find who or what is holding it.

**Comment by AnturK** [comment]
get_atom_on_turf(thing) recurses from the object location up to turf or given type. I guess the name might be bit misleading.

**Pull request merged**